### PR TITLE
Change 'REPLConsole.session' to store only the latest single session

### PR DESCRIPTION
**Pull request by sh19910711**
Make a change of `REPLConsole.session` to store only the latest single session, since the current way does not work properly after the console element is closed.

### CODE REVIEW

File: lib/web_console/templates/console.js.erb [hunk 1]
```diff
@@ -147,6 +147,8 @@ REPLConsole.prototype.install = function(container) {
   findChild(container, 'resizer').addEventListener('mousedown', resizeContainer);
   findChild(consoleActions, 'close-button').addEventListener('click', closeContainer);
   consoleOuter.addEventListener('DOMNodeInserted', shiftConsoleActions);
+
+  REPLConsole.current_session = this;
```
Comment (gsamokovarov) [inline lib/web_console/templates/console.js.erb]: Can we name this in camelCase? Seems to be the convention in JavaScript.

Comment (sh19910711) [inline lib/web_console/templates/console.js.erb]: Oh, thanks for noticing that. I just updated it.

File: lib/web_console/templates/console.js.erb [hunk 2]
```diff
@@ -409,12 +411,13 @@ REPLConsole.installInto = function(id) {
...
   replConsole.install(consoleElement);
-  REPLConsole.session[remotePath] = replConsole;
   return replConsole;
 };
 
-// Store instances with the remote paths
-REPLConsole.session = {};
+// This is to store the latest single session, and the stored session
+// is updated by the REPLConsole#install() method.
+// It allows to operate the current session from the other scripts.
```
Comment (gsamokovarov) [inline lib/web_console/templates/console.js.erb]: When you say other scripts, you mean scripts like `error_page.js.erb`?

Comment (sh19910711) [inline lib/web_console/templates/console.js.erb]: > When you say other scripts, you mean scripts like error_page.js.erb?

Yes, I do. Do you have better words for it?

Comment (gsamokovarov) [inline lib/web_console/templates/console.js.erb]: Nah, was just wondering. :-)

**Comment by gsamokovarov** [comment]
If that was a bug, we can add a regression test for it. @sh19910711 did you found or are you aware any other issues? If not, we can release a new version soon.

**Comment by sh19910711** [comment]
> If that was a bug, we can add a regression test for it.

OK. I think it's all right :)

**Comment by gsamokovarov** [comment]
@sh19910711 Is this good to go?

**Comment by sh19910711** [comment]
@gsamokovarov Yeah, this is. Thanks.

On June 24, 2015, at 5:37 AM, Genadi Samokovarov <notifications@github.com> wrote:

@sh19910711 Is this good to go?

**Comment by sh19910711** [comment]
@gsamokovarov Thanks for merging :octocat:

**Comment by gsamokovarov** [comment]
Thanks for working on this. :-)

References: #121. I do this, so we can track all the GSoC issues we fixed this summer.

**Pull request merged**